src/main/java/com/mycompany/exercisetwo/Death.java

```diff
@@ -0,0 +1,15 @@
+package com.mycompany.exercisetwo;
+
+import java.time.LocalDate;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public interface Death {
+
+    public String die(LocalDate born, LocalDate die, String deathCause);
```

Comment:
The exercise asked for it but I like the fact you use LocalDate for years, so I'm happy with this approach 😄